### ISSUE TITLE
Talkback does not announce long press action on TouchableNativeFeedback

### ISSUE THREAD

**Issue by sindujaramaraj** [facebook/react-native#32229]
## Description
On TouchableNativeFeedback element, TalkBack only announces "Double tap to activate" and not "Double tap to activate and hold for long press"

## React Native version:
0.62.2
(although this seems to be an issue on the latest version as well)

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. Create a TouchableNativeFeedback with onPress and onLongPress
2. Observer what TalkBack announces

## Expected Results
RN components allowing long press gestures must announce "Double tap to activate and hold for long press"

## Snack, code example, screenshot, or link to a repository:
https://snack.expo.dev/@siramara/accessibility-test-on-touchablenativefeedback
